_docs/style/interactive-states.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Interactive States
+layout: docs
+category: Style
+usage: >
+  The system of interactive states ensures consistent styles for the states of controls in Helix. This system accounts for multiple types of controls, with various interactive patterns, and defines commonalities and standards across all of them.
+
+  All controls that live in the application body strictly adhere to subsets of the specifications in this document, unless otherwise noted in their individual design docs.
+resource: true
+status: complete
+last-modified: 2019-06-03
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use the interactive states styles when you design a control or input that you place within the body of the application. Consistent use of interactive state styles ensure that your design is aligned and consistent with Helix.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct interactive states:
+
+- In general, only apply hovered states to selectable controls. For example, checkboxes and radios have a hover state, but text inputs do not.
+- Do not apply all states styles to your inputs or controls.
+- When you design a new control, consult other similar controls to understand which interactive states apply.
+- Avoid using selected filled styles for large controls, such as tiles or cards.
+- If you have an input or control that is selected and errored, consider if there is a better interaction design that prevents the user from getting into that state.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Specifications
+
+Use the following specifications when you construct an interactive UI element.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+In this document, a circle within a square represents a state style.
+
+An interactive state is comprised of the following elements:
+
+- **Control**: The control is the container of the UI component to which you apply the interactive state.
+- **Emphasis color**: Apply the emphasis color to a primary child element within a control.
+
+The following list of elements receive emphasis color:
+
+- Checks in checkboxes
```

Comment:
Style guide uses check boxes and radio buttons rather than checkboxes and radios. Let me know if I need to update the Style Guide.

Reply:
@cyrichardson , yes - Helix uses "checkboxes" and "radios" (rather than radio buttons).